dags/glam_imports.py

```diff
@@ -0,0 +1,119 @@
+"""
+Desktop ETL for importing glam data into GLAM app
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from operators.gcp_container_operator import GKENatPodOperator
+from operators.task_sensor import ExternalTaskCompletedSensor
+from airflow.models import Variable
+from airflow.operators.subdag_operator import SubDagOperator
+
+from glam_subdags.extract import extracts_subdag, extract_user_counts
+from glam_subdags.histograms import histogram_aggregates_subdag
+from glam_subdags.general import repeated_subdag
+from glam_subdags.generate_query import generate_and_run_desktop_query
+from utils.gcp import bigquery_etl_query, gke_command
+from utils.tags import Tag
+
+default_args = {
+    "owner": "akommasani@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 10, 22),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "akommasani@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+}
+
+
+
+tags = [Tag.ImpactTier.tier_2]
+
+dag = DAG(
+    "glam_imports",
+    default_args=default_args,
+    schedule_interval="0 7 * * *",
+    doc_md=__doc__,
+    tags=tags,
+)
```

Comment:
I guess there is the risk of the GLAM DAG actually running longer than 5 hours and data not being entirely up-to-date when this DAG gets executed. Might be worth adding a comment to add this caveat here and maybe explain why this needs to be done.